packages/cli/src/oclif/commands/scaffold.js

```diff
@@ -0,0 +1,217 @@
+        lines.splice(0, 0, importerLine);
+
+        // insert '[Resource.key]: Resource,' after 'resources:' line
+        const injectAfter = `${typeMap[type]}: {`;
+        const injectorLine = `[${varName}.key]: ${varName},`;
+        const linesDefIndex = lines.findIndex(line =>
+          line.endsWith(injectAfter)
+        );
+
+        if (linesDefIndex === -1) {
+          this.stopSpinner(false);
+          this.log();
+          this.log(
+            colors.bold(`Oops, we could not reliably rewrite your ${entry}.`) +
+              ' Please add:'
+          );
+          this.log(` * \`${importerLine}\` to the top`);
+          this.log(
+            ` * \`${injectAfter} ${injectorLine} },\` in your app definition`
+          );
+          return Promise.resolve();
+        } else {
+          lines.splice(linesDefIndex + 1, 0, '    ' + injectorLine);
+          return utils
+            .writeFile(entryFile, lines.join('\n'))
+            .then(this.stopSpinner);
+        }
+      })
+      .then(() =>
+        this.log(
+          '\nFinished! We did the best we could, you might gut check your files though.'
+        )
+      )
+      .catch(message => {
+        this.stopSpinner(false);
+        this.log(colors.red(`We couldn't scaffold your files:`));
+        this.log(message);
+      });
+  }
+}
+
+ScaffoldCommand.args = [
+  {
+    name: 'type',
+    help: 'what type of thing are you creating',
+    required: true,
+    options: ['resource', 'trigger', 'search', 'create']
+  },
+  {
+    name: 'name',
+    help: 'the name of the new thing to create',
+    required: true
+  }
+];
+
+ScaffoldCommand.flags = buildFlags({
+  commandFlags: {
+    dest: flags.string({
+      description:
+        "Sets the new file's path. The default pattern is {type}s/{name}"
+    }),
+    entry: flags.string({
+      description: 'Where to import the new file',
+      default: 'index.js'
+    }),
+    force: flags.boolean({
+      char: 'f',
+      description: 'Should we overwrite an exisiting file',
+      default: false
+    })
+  }
+});
+
+ScaffoldCommand.examples = [
+  'zapier scaffold resource "Contact"',
+  'zapier scaffold resource "Contact" --entry=index.js',
+  'zapier scaffold resource "Contag Tag" --dest=resources/tag',
+  'zapier scaffold trigger "Existing Create" --force'
+];
+
+ScaffoldCommand.description = `Adds a starting resource, trigger, action or search to your app.
+
+The first argument should one of \`resource|trigger|search|create\` followed by the name of the file.
+
+The scaffold command does two general things:
+
+* Creates a new destination file like \`resources/contact.js\`
+* (Attempts to) import and register it inside your entry \`index.js\`
+
+You can mix and match several options to customize the created scaffold for your project.
+
+> Note, we may fail to rewrite your \`index.js\` so you may need to handle the require and registration yourself.
```

Comment:
`handle the require and registration yourself` is pretty harsh laguage (makes it sound very hard). Might replace with "paste the import line yourself" or something.

Reply:
Yeah. Copy/paste from the old command. I'll go through the language.